src/components/AboutProject/AboutProject.jsx

```diff
@@ -0,0 +1,52 @@
+import styles from './AboutProject.module.css';
+import image1 from '../../assets/svg/Frame1.svg';
+import image2 from '../../assets/svg/Frame2.svg';
+import image3 from '../../assets/svg/Frame3.svg';
+import arrow from '../../assets/svg/arrow-right.svg';
+
+function AboutProject() {
+  return (
+    <div id="about" className={styles.about}>
+      <div className={'container'}>
```

Comment:
```suggestion
      <div className="container">
```
обычно пишут так.
а с styles уже будет 
`<div className={styles.yourClass}>`

исправь везде